### DISCUSSION TITLE
Option to dismiss stale approvals only for non-owners

### DISCUSSION THREAD

**Discussion by xanderman** [community/community#36801] [Pull Requests]
My security team really wants us to enable automatically dismissing stale approvals. I find this setting to be an enormous drain on productivity. "Approval with small comments" is a very common case for things like typos or lint issues, and this setting completely thwarts this flow by requiring us to re-review after even the tiniest of changes. The problem is, there's no middle ground. So I'd like to propose one.

People in CODEOWNERS are generally people I trust not to abuse approvals. They care about the code and have been "vetted", so to speak, by the team. I would love an option that scopes automatic dismissal of approval to non-owners. I'm not sure if I think that "non-owners" should refer to PR authors or commit authors, but probably the latter. If the added commits are from a code owner, don't dismiss stale approvals.

**Comment by ossanna16** [comment]
Thank you for asking a good question @xanderman, I'm glad you found our community 🙂
